app/util/file.ts

```diff
@@ -0,0 +1,7 @@
+import * as fs from 'fs';
```

Comment:
Use the fs/promises API. Another file in this PR is using it as well.  https://nodejs.org/api/fs.html#fs_promises_api